### PR TITLE
libiodbc, add libVersionCompat, disable and remove static library

### PR DESCRIPTION
**Pull request by Begasus**

### CODE REVIEW

**Comment by Begasus** [comment]
@waddlesplash this one produced a static library (libodbc.a) and shared library (libodbc.so) without an error from haikuporter, a flaw here?

File: dev-libs/libiodbc/libiodbc-3.52.12.recipe [hunk 1]
```diff
@@ -49,14 +52,21 @@ BUILD_PREREQUIRES="
 
 BUILD()
 {
-	runConfigure ./configure --with-iodbc-inidir=$settingsDir/etc
+	runConfigure ./configure --with-iodbc-inidir=$settingsDir/etc \
+		--disable-static
```
Comment (korli) [inline dev-libs/libiodbc/libiodbc-3.52.12.recipe:56]:
```suggestion
		--disable-static --disable-libodbc
```

**Comment by korli** [comment]
Could be bumped to https://github.com/openlink/iODBC/archive/refs/tags/v3.52.15.tar.gz

File: dev-libs/libiodbc/libiodbc-3.52.12.recipe (hunk 1 above)
Comment (Begasus) [reply dev-libs/libiodbc/libiodbc-3.52.12.recipe:56]: OK, could add this suggestion in the browser, but I know I would mess up getting things on par. Checking. :)

**Comment by korli** [comment]
Sure. Just suggesting, when only the minor version number is incremented, it shouldn't be a problem to build correctly :)

**Comment by Begasus** [comment]
Leaving as wip for a while so @waddlesplash can investigate :)

**Comment by waddlesplash** [comment]
Well, without seeing an exact version of the recipe with the problem, there isn't much for me to investigate.

**Comment by Begasus** [comment]
> Well, without seeing an exact version of the recipe with the problem, there isn't much for me to investigate.

As mentioned on IRC, you can build the current one in haikuports that still has the old changes and libodbc in prepareInstalledDevelLibs, only need to add --disable-static for the default libraries, no need to check this, here it is fixed.

**Pull request merged**